cpp/log/etcd_consistent_store-inl.h

```diff
@@ -448,15 +457,15 @@ util::Status EtcdConsistentStore<Logged>::CreateEntry(const std::string& path,
 
 template <class Logged>
 template <class T>
-util::Status EtcdConsistentStore<Logged>::ForceSetEntry(
-    const std::string& path, EntryHandle<T>* t) {
+util::Status EtcdConsistentStore<Logged>::ForceSetEntry(EntryHandle<T>* t) {
   CHECK_NOTNULL(t);
-  CHECK(!t->HasHandle());
```

Comment:
I'd convinced myself that it wasn't necessary, but then we spoke about it offline and you reminded me of my arguments for having it there when I put it in, so I've re-instated it and added a comment explaining why it's there [to remind me, at least, next time I go to delete it.]